mqtt3/1.0.0/modules/ROOT/pages/mqtt3-connector-listener.adoc

```diff
@@ -0,0 +1,121 @@
+
+[source,xml,linenums]
+----
+    <flow name="listenerArgentinaTemperature">
+        <mqtt3:listener config-ref="${config}">
+            <mqtt3:topics>
+                <mqtt3:topic topicFilter="quotes/england/#"/>
+            </mqtt3:topics>
+        </mqtt3:listener>
+
+        <logger level="DEBUG" message="A quote has been published to #[attributes.topic]: #[payload]"/>
+    </flow>
+----
+
+Messages published to the following topics trigger the listener:
+
+* `quotes/england/authors/terryPratchett`
+* `quotes/england/authors/neilGaiman`
+* `quotes/england/actors`
+
+However, messages published to the following topics do not trigger the listener:
+
+* `quotes/american/actors`
+* `phrases/england/authors/neilGaiman`
+
+== Listeners Sharing a Configuration
+
+Listeners that share a configuration element also share a connection and client ID. This is important to keep in mind especially in the cases where one of the listeners is stopped but the other isn't.
+
+Suppose we have the following configuration element and listeners in our application:
+
+In the following example, the listeners of the flows `listenerReaderA` and `listenerReaderB` share the same connection `MQTT_Config` configuration and client ID `smart-bentley-123`. From the broker's perspective both listeners are indistinguishable:
+
+[source,xml,linenums]
+----
+<mqtt3:config name="MQTT_Config">
+    <mqtt3:connection clientId="smart-bentley-123" url="tcp://127.0.0.1:1883" />
+</mqtt3:config>
+<flow name="listenerReaderA">
+   <mqtt3:listener config-ref="MQTT_Config">
+            <mqtt3:topics>
+                <mqtt3:topic topicFilter="shakespeare"/>
+                <mqtt3:topic topicFilter="terryPratchett"/>
+            </mqtt3:topics>
+   </mqtt3:listener>
+   <logger level="INFO"  message="Received message '#[payload]' with at topic #[attributes.topic] with qos #[attributes.qos]">
+</flow>
+<flow name="listenerReaderB">
+    <mqtt3:listener config-ref="MQTT_Config">
+            <mqtt3:topics>
+                <mqtt3:topic topicFilter="neilGaiman"/>
+                <mqtt3:topic topicFilter="terryPratchett"/>
+            </mqtt3:topics>
+   </mqtt3:listener>
+   <logger level="INFO"  message="Received message '#[payload]' with at topic #[attributes.topic] with qos #[attributes.qos]">
+</flow>
+----
+
+As you can see from the XML example, the listeners also share a subscription to the topic `terryPratchett`. Whichever listener subscribes first to the topic can set the subscription quality of service (QoS). Only one subscription with one quality of service level can exist for that topic.
+
+The second observation is that, if the `listenerReaderB` flow stops, the `listenerReaderA` flow still receives and processes the messages for the `terryPratchett` topic.  Even if the configuration element specifies a `cleanSession=false`, the messages that were processed while `listenerReaderB` was offline, are not resent to this listener.
```

Comment:
I suggest removing "The second observation is that" and start with "If the...." It makes the text very wording and is not needed.

Reply:
```suggestion
The second observation is that, if the `listenerReaderB` flow stops, the `listenerReaderA` flow still receives and processes the messages for the `terryPratchett` topic.  Even if the configuration element specifies `cleanSession=false`, the messages that were processed while `listenerReaderB` was offline, are not resent to this listener.
```